eoscostarica/guias.eoscostarica.io
# feat(frontend): add demux pattern page

### Add demux pattern page

### What does this PR do?

- resolves [#444](https://github.com/eoscostarica/guide.eoscostarica.io/issues/444)

### Steps to test
1. Go to "Blockchain EOSIO"
1. Open "Patrón Demux"
1. Check the content and metadata

#### CheckList
